docs/reference/support-policy.md

```diff
@@ -102,11 +102,11 @@ compatibility tables to describe the interoperability requirements of these piec
 
 ### JavaScript Runtime
 
-| Stencil Version | Node v10 | Node v12 | Node v14 | Node v16 | Node v18  |  Deno*  |
-|:---------------:|:--------:|:--------:|:--------:|:--------:|:---------:|:-------:|
-|       V3        | &#10060; | &#10060; | &#9989;  | &#9989;  |  &#9989;  | &#9888; |
-|       V2        | &#10060; | &#9989;  | &#9989;  | &#9989;  | &#9888;** | &#9888; |
-|       V1        | &#9989;  | &#9989;  | &#9989;  | &#9989;  | &#10060;  | &#9888; |
+| Stencil Version | Node v10 | Node v12 | Node v14 | Node v16 | Node v18  |  Node v20 |  Deno*  |
+|:---------------:|:--------:|:--------:|:--------:|:--------:|:---------:|:---------:|:-------:|
+|       V3        | &#10060; | &#10060; | &#9989;  | &#9989;  |  &#9989;  | &#9989;   | &#9888; |
+|       V2        | &#10060; | &#9989;  | &#9989;  | &#9989;  | &#9888;** | &#10060;  | &#9888; |
+|       V1        | &#9989;  | &#9989;  | &#9989;  | &#9989;  | &#10060;  | &#10060;  | &#9888; |
```

Comment:
I figured we just treat v3 as one thing here, but should we make a separate row for v3.3 here instead?

Reply:
When we added Node 18 support, that was slated for v3.1 - https://github.com/ionic-team/stencil-site/pull/955. I think leaving this at V3 for Node 20 is fine 👍

Reply:
When can we remove the Deno column?

Reply:
> When can we remove the Deno column?

I think we can remove that for/with the v4 release. I'll add a ticket to JIRA

Reply:
STENCIL-835